lastpass/README.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+
+[LastPass][1] is a password management solution that securely stores and manages passwords and other sensitive
+information. LastPass provides users with the ability to generate passwords, synchronize passwords across multiple
+devices, and add an extra layer of security through multi-factor authentication.
+
+Integrate with Datadog to gain insights into reporting event logs through the LastPass Enterprise API. The data will be
+normalized and enriched before ingestion. Pre-built dashboard visualizations provide immediate insights into LastPass
+reporting events, while out-of-the-box detection rules enhance detection and response capabilities.
+
+## Setup
+
+### Configuration
+
+#### Get config parameters of LastPass
+
+##### Account Number
+
+1. Use your email address and master password to log in and access the new [Admin Console](https://admin.lastpass.com/).
+2. On the Dashboard tab, click the Profile email located in the top right corner to find the Account Number, which is
+   preceded by the words "Account number".
+3. Alternatively, you can find the Account Number, preceded by the words "Account number," by navigating to Advanced >
+   Enterprise API.
+
+##### Provisioning Hash
```

Comment:
@maycmlee Thanks for your feedback here.
As this word **Provisioning Hash** is a title for user input for installing a integration, we have kept **Hash** instead of **hash**. 
Please let us know your thoughts.